mmdet/models/dense_heads/yolo_head.py

```diff
@@ -147,6 +148,22 @@ def _init_layers(self):
             self.convs_bridge.append(conv_bridge)
             self.convs_pred.append(conv_pred)
 
+    def init_weights(self):
+        for m in self.modules():
+            if isinstance(m, nn.Conv2d):
+                normal_init(m, mean=0, std=0.01)
+            if is_norm(m):
+                constant_init(m, 1)
+
+        # Use prior in model initialization to improve stability
+        for conv_pred, stride in zip(self.convs_pred, self.featmap_strides):
+            bias = conv_pred.bias.reshape(self.num_anchors, -1)
```

Comment:
Is this line necessary?

Reply:
The origin shape of the bias is [255]. It would be easier to initialize each anchor's classification bias and objectness bias after reshape to [3, 85]. Otherwise, the index will be complicated.